src/slc/outdated/browser.py

```diff
@@ -31,7 +32,7 @@ def toggle(self, value=None):
         event = ObjectOutdatedToggleEvent(self.context, self.outdated)
         notify(event)
         name = self.context.title_or_id()
-        if not isinstance(name, unicode):
+        if not isinstance(name, six.text_type):
             name = name.decode(getSiteEncoding(self.context))
```

Comment:
I would use `safe_unicode` and drop the deprecated `getSiteEncoding`.

Reply:
Seems reasonable.